docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -36,12 +36,16 @@ Click **Create GitHub App** to finish.
 3. Select **Only select repositories**, and choose the repository to sync with Web Modeler.
 4. Once redirected to your application's installation page, copy the **Installation ID** located at the end of the page's URL: `https://github.com/settings/installations/{installation_id}`.
 
-### Configure GitHub in Web Modeler
+### [Configure GitHub in Web Modeler](#configure-github-in-web-modeler)
 
 :::note
 An organization administration account (or project administrator in Camunda Self-Managed) is required for the initial GitHub configuration.
 :::
 
+:::note
+When using a self-hosted GitHub instance, ensure the environment variable `CAMUNDA_MODELER_GITSYNC_GITHUB_BASEURL` is set to the API URL of your self-hosted GitHub instance. It usually looks like `http(s)://HOSTNAME/api/v3`. Refer to [GitHub documentation](https://docs.github.com/en/enterprise-server@3.15/rest/enterprise-admin?apiVersion=2022-11-28#endpoint-urls) and choose the correct enterprise server version.
+:::
+
```

Comment:
I'm wondering if this might be more apparent to a reader if included as part of the steps? This may make it more visible vs two back-to-back `:::note` blocks. For example:

```suggestion
1. To use a **self-hosted GitHub instance,** set the environment variable `CAMUNDA_MODELER_GITSYNC_GITHUB_BASEURL` to the API URL of your self-hosted GitHub instance (for example, `http(s)://HOSTNAME/api/v3`). Refer to the [GitHub documentation](https://docs.github.com/en/enterprise-server@3.15/rest/enterprise-admin?apiVersion=2022-11-28#endpoint-urls) and choose the correct Enterprise Server version.
```

Also a small wording suggestion.

Reply:
Mhm, good point, but I don't know if I prefer it. @marcellobarile, any opinion here as the DRI?

Reply:
> I'm wondering if this might be more apparent to a reader if included as part of the steps

To be noted that it's not a mandatory step. Could this better drive the decision?

Reply:
I will leave it as is for now and revisit later when GitLab is added.